sphinx_fossasia_theme/static/fossasia.css_t

```diff
@@ -117,4 +117,10 @@ a {
 .highlighted {
     background-color: #FFEB3B;
     border-radius: 10px;
-}
\ No newline at end of file
+}
+
+img {
```

Comment:
Sorry if this question seems naive, but is this enough for all img tags to act as img-responsive?